{"name": "获取同步根组id", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <bms.testcases.test_02_usersync.TestUserSync object at 0x000002A712CE69D0>\ncaseid = 'usersync_get_syncgroupid_01'\ncasedata = {'base_url': '${config(host)}', 'caseid': 'usersync_get_syncgroupid_01', 'description': '获取同步根组id', 'epic': '用户管理', ...}\n\n    @pytest.mark.parametrize('caseid, casedata', [(caseid, casedata) for caseid, casedata in\n                                                  readcase.ReadCase().get_case_dict(\n                                                      ROOT_DIR + 'bms/data/usersync/usersync_get_syncgroupid.yaml').items()])\n    def test_usersync_get_syncgroupid(self, caseid, casedata):\n        handleallure.allure_display(casedata)\n        logging.info(\"==========开始执行用例：%s==========\", caseid)\n>       runcase.excute_case(casedata)\n\nbms\\testcases\\test_02_usersync.py:165: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\ncommon\\runcase.py:54: in excute_case\n    checkresult.check_result(hope_result, recv_data, recv_code)\ncommon\\checkresult.py:30: in check_result\n    assert_response(value, real_res)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nhope_res = {'msg': 'success', 'statusCode': 0}\nreal_res = {'msg': '找不到对应的组织机构', 'statusCode': 1114886}\n\n    def assert_response(hope_res, real_res):\n        \"\"\"\n        返回结果校验 -全返回校验\n        :param hope_res:\n        :param real_res:\n        :return:\n        \"\"\"\n        try:\n            with allure.step(\"返回值校验\"):\n                allure.attach(name=\"期望返回值\", body=str(hope_res))\n                allure.attach(name='实际返回值', body=str(real_res))\n                if isinstance(hope_res, (dict, list)) and isinstance(real_res, (dict, list)):\n                    flag = handledict.cmp_dict(hope_res, real_res)\n>                   assert flag\nE                   AssertionError\n\ncommon\\checkresult.py:45: AssertionError"}, "description": "获取同步根组id", "steps": [{"name": "执行关联接口", "status": "passed", "steps": [{"name": "执行当前接口：同步任务列表搜索-无搜索条件", "status": "passed", "attachments": [{"name": "请求方法：", "source": "f570a60b-c635-4109-9c96-64a08eb3c721-attachment.attach"}, {"name": "请求地址", "source": "6faaca47-d28e-4676-9450-efb9c1307664-attachment.attach"}, {"name": "请求头", "source": "3e71340a-810a-4af1-95a8-e822ba20bf64-attachment.attach"}, {"name": "请求参数", "source": "cec24be2-76ba-41de-9ec6-3af75401b0a9-attachment.attach"}, {"name": "请求结果", "source": "ff8f3b73-ac8f-4194-bf96-5c2b9fc951b9-attachment.attach"}], "start": 1687773648107, "stop": 1687773648234}, {"name": "结果校验", "status": "passed", "steps": [{"name": "状态码校验", "status": "passed", "attachments": [{"name": "期望状态码", "source": "ebefb160-8549-4f92-b965-8cf62b2516d0-attachment.attach"}, {"name": "实际状态码", "source": "c6b9592f-2593-4b8f-822c-c27042cfe906-attachment.attach"}], "start": 1687773648234, "stop": 1687773648238}, {"name": "返回值校验", "status": "passed", "attachments": [{"name": "期望返回值", "source": "8b3d8989-2335-43a2-a9e1-8dfa49f7412c-attachment.attach"}, {"name": "实际返回值", "source": "e763f0a6-fc85-43a9-be1a-77e93c69f1c1-attachment.attach"}], "start": 1687773648238, "stop": 1687773648241}], "start": 1687773648234, "stop": 1687773648241}], "start": 1687773648100, "stop": 1687773648246}, {"name": "执行当前接口：获取同步根组id", "status": "passed", "attachments": [{"name": "请求方法：", "source": "da5cb332-3593-4b18-be85-b89d4cc87531-attachment.attach"}, {"name": "请求地址", "source": "4c1ae38c-8032-4301-a04c-ce537f1ff100-attachment.attach"}, {"name": "请求头", "source": "575b863a-f16a-4d05-ac98-cb405f654f34-attachment.attach"}, {"name": "请求参数", "source": "427565d2-d004-4a73-b19e-f59376620a18-attachment.attach"}, {"name": "请求结果", "source": "44f18121-07a8-4480-80c9-efb68a47e21f-attachment.attach"}], "start": 1687773648248, "stop": 1687773648317}, {"name": "结果校验", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"E:\\Wondersoft_ApiTest\\common\\checkresult.py\", line 30, in check_result\n    assert_response(value, real_res)\n  File \"E:\\Wondersoft_ApiTest\\common\\checkresult.py\", line 45, in assert_response\n    assert flag\n"}, "steps": [{"name": "状态码校验", "status": "passed", "attachments": [{"name": "期望状态码", "source": "a804bc09-981b-4d04-801c-497ba7f215b9-attachment.attach"}, {"name": "实际状态码", "source": "b3b5785a-bbb6-4c63-bf6b-64707c06499b-attachment.attach"}], "start": 1687773648317, "stop": 1687773648318}, {"name": "返回值校验", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"E:\\Wondersoft_ApiTest\\common\\checkresult.py\", line 45, in assert_response\n    assert flag\n"}, "attachments": [{"name": "期望返回值", "source": "3a17a90e-91ff-493a-90fa-b5522004ed9a-attachment.attach"}, {"name": "实际返回值", "source": "a0650a41-e8f1-44f9-84e7-e849218d6e00-attachment.attach"}], "start": 1687773648318, "stop": 1687773648319}], "start": 1687773648317, "stop": 1687773648370}], "attachments": [{"name": "log", "source": "0ea42f69-1856-4430-94e0-aa5f0fdc4125-attachment.txt", "type": "text/plain"}], "start": 1687773648099, "stop": 1687773648371, "uuid": "a881f2c7-24db-4a60-aa34-02283fc8b201", "historyId": "c076d0bb657b7fadcb6dc7f1415a3243", "testCaseId": "8a41af8eca098c4295d5e872d129839c", "fullName": "bms.testcases.test_02_usersync.TestUserSync#test_usersync_get_syncgroupid", "labels": [{"name": "epic", "value": "用户管理"}, {"name": "feature", "value": "用户同步管理-用户同步"}, {"name": "story", "value": "编辑，获取同步根组id接口"}, {"name": "parentSuite", "value": "bms.testcases"}, {"name": "suite", "value": "test_02_usersync"}, {"name": "subSuite", "value": "TestUserSync"}, {"name": "host", "value": "OFYI7JX9DURTDV6"}, {"name": "thread", "value": "16008-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "bms.testcases.test_02_usersync"}]}